src/govuk/components/accordion/accordion.mjs

```diff
@@ -34,7 +34,7 @@ function Accordion ($module) {
       showSection: 'Show<span class="govuk-visually-hidden"> this section</span>'
     }
   }
-  this.config = mergeConfigs(defaultConfig, $module.dataset)
+  this.config = mergeConfigs(defaultConfig, config || {}, $module.dataset)
```

Comment:
Could we expand the existing config merging tests to make sure it handles empty objects ok?

Reply:
I've added a new unit test which runs `mergeConfigs` with empty objects at various positions and compares those with the result of the same function sans-empty objects. Do you think that covers it?

Reply:
I think that covers it, yep 👍